doc/api/fs.md

```diff
@@ -3572,8 +3572,8 @@ console.log(fs.readFileSync('temp.txt', 'utf8'));
 // Prints: Node.js
 
 async function doTruncate() {
-  const fd = await fsPromises.open('temp.txt', 'r+');
-  await fsPromises.ftruncate(fd, 4);
+  const filehandle = await fsPromises.open('temp.txt', 'r+');
+  await filehandle.truncate(4);
   console.log(fs.readFileSync('temp.txt', 'utf8'));  // Prints: Node
```

Comment:
these should really be....

```js
let filehandle;
try {
  const filehandle = await fsPromises.open('temp.txt', 'r+');
  await filehandle.truncate(4);
} finally {
  await filehandle.close();
}
```

While the `filehandle` will close automatically on garbage collection in order to prevent the leak, it *should* be closed manually. A process warning would be emitted if it is allowed to close on gc.

Reply:
@jasnell - what do you think the right avenue to discuss such a cross-cutting concern is? I'm not sure what the right team/working-group/avenue is.

I'm a little lost here :)